Resources/input/fsh/examples/UC1/UC1_ServiceRequests.fsh

```diff
@@ -1,98 +1,117 @@
 // Prescription Service Requests
 Instance: UC1-Initial-Prescription-Request
 InstanceOf: ERPServiceRequestPrescriptionRequest
-Usage: #inline
+Usage: #example
 Title: "Initial Prescription Request"
 Description: "This ServiceRequest is sent initially to the prescribing practitioner"
-* identifier[0]
-  * system = "https://gematik.de/fhir/erp-servicerequest/sid/NamingSystemRequestIdentifier"
-  * value = "1"
-* identifier[+].system = "https://gematik.de/fhir/erp-servicerequest/sid/NamingSystemPreDisIdentifier"
-* identifier[=].value = "PreDis-1"
+* identifier[requestId].value = "1"
```

Comment:
Warum gibt es hier keinen https://gematik.de/fhir/erp-servicerequest/sid/ProcedureIdentifier
mehr?

Reply:
Scheint nur in diesem Beispiel nicht gepflegt zu sein.

Reply:
Der ProcedureIdentifier ist unter requisition zu finden. Da es nur einen Wert für die requisition gibt sieht man das in Zeile 9